docs/vocs/docs/pages/book/advanced-usage/sdk.mdx

```diff
@@ -81,35 +91,46 @@ OpenVM supports generating three types of proofs. The sections below describe ho
 
 ### Generating App Proofs
 
-After building and transpiling a program, you can then generate a proof. To do so, you need to commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
+After building and transpiling a program, you can then generate a proof. To do so, you need to convert your ELF into a `VmExe`, commit your `VmExe`, generate an `AppProvingKey`, format your input into `StdIn`, and then generate a proof.
+
+The SDK, however, combines the first few steps for you. Create a prover and pass in your input as a `StdIn` to generate an app proof.
```

Comment:
I think we should say somewhere what the different `ExecutableFormat`s you can put into the prover constructor are: this is relevant for all 3 prover types